_includes/selector.html

```diff
@@ -483,6 +495,27 @@
 
                 return lines;
             },
+            getpipCmdHtml() {
+                var pip_install = "pip install ";
+                var index_url = " --extra_index_url=https://pypi.nvidia.com";
+                var pkgs = String(this.active_packages).replaceAll(",", "-cu11 ").toLowerCase() + "-cu11";
```

Comment:
```suggestion
                var libraryToPkg = (pkg) => {
                    pkg = pkg.toLowerCase();
                    if (pkg === "cucim") return pkg;
                    return pkg + "-cu11"
                }
                var pkgs = this.active_packages.map(libraryToPkg).join(" ");
```

Quick suggestion to use a helper function to rename the libraries to their respective `wheel` package names.

This is a bit safer than casting the array to a string with `String()`.

It will also let us remove the `if (pkgs.includes("cucim"))` block below.